grails-app/i18n/messages.properties

```diff
@@ -4153,3 +4153,8 @@ react.productsList.column.dateCreated.label=Created on
 # React table common
 react.reactTable.pagination.of.label=of
 react.reactTable.pagination.rows.label=rows
+
+# React new features bar
+react.newFeature.autosave.version.beta.label=BETA
+react.newFeature.readMore.label=Read more
+react.newFeature.autosave.title.label=New autosave feature is here!
```

Comment:
This `newFeature` sounds a little bit different for me. Additionally I think that `BETA` label is not only for `autosave`, so maybe we should remove this part of label? My suggestion is something similar to `react.bar.version.beta.label` What do you think @awalkowiak?

Reply:
yeah, you are right it shouldn't have `.autosave`, it should be `react.newFeature.version.beta.label`.
As for `newFeature` I will leave it for you, I have not managed to come up with anything more reasonable.